repository-management.md

```diff
@@ -0,0 +1,119 @@
+# Repository Management
+
+## Package Names
+
+Package names for the official `wp-cli` GitHub repository need to be discussed upfront with the [maintainers](https://github.com/orgs/wp-cli/teams/maintainers).
+
+Before creating a new repository, it's important to discuss the repository first. This discussion helps establish consensus on what purpose the codebase serves, and where it lives best in the WP-CLI ecosystem.
+
+Package names for WP-CLI commands must end with the `-command` suffix for clarity, and also to distinguish them from the non-command repositories.
+
+Before new repositories are created for commands, a decision needs to be made whether the command deserves its own repository or whether it should be included within an existing repository instead. 
+
+_Example: `wp-cli/scaffold-package-command`._
+
+## Package Versions
+
+Package versions follow the [Semantic Versioning Specification (SemVer)](http://semver.org/).
+
+In layman's terms for us, this typically means:
+
+* *Patch releases* (e.g. 1.0.x) are used for bug fixes, documentation changes, small enhancements, etc.
+* *Minor releases* (e.g. 1.x.0) are used when a new command is introduced to a package, or a major enhancement is made to an existing command.
+* *Major releases* (e.g. x.0.0) are used when changes to the framework change the WP-CLI API in a breaking way.
+
+## Milestones
+
+Open milestones usually point to the next version to be released.
+
+As the release notes are built using the milestones, both pull requests and issues that are to be merged need to be set to the upcoming milestone, so that these pull requests will be part of the release notes.
+
+## Labels
+
+Labels are an important organizational tool to communicate the state and progress of the work being done. [Committers](https://github.com/orgs/wp-cli/teams/committers) are required to keep these up-to-date.
```

Comment:
Might be worth saying something along the lines of:

> Keeping issues and pull requests accurately labeled helps us make sense of issue tracker usage in aggregate and more easily search closed issues and pull requests for past discussion.

Reply:
e.g. better explaining _why_

Reply:
https://github.com/wp-cli/handbook/pull/127/commits/31e75dd539e5f045e97c081247709ed9a651ac77